src/Akka.Persistence.Azure/Journal/AzureTableStorageJournal.cs

```diff
@@ -235,17 +220,16 @@ protected override void PreStart()
         {
             _log.Debug("Initializing Azure Table Storage...");
 
-            // forces loading of the value
-            var name = Table.Name;
+            InitCloudStorage(5)
+                .ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Had to do it this way since we don't have an async actor `PreStart` and `PostStop`

Reply:
LGTM